tests/WE2E/get_WE2Etest_names_subdirs_descs.sh

```diff
@@ -1051,17 +1050,17 @@ configuration files of the primary WE2E tests...
 #    to cover those cases.
 #
 # 2) The double quotes (which need to be escaped here, i.e. \") are needed
-#    so that for any experiment variables that are arrays, all the elements 
-#    of the array are combined together and treated as a single element.  
-#    If the experiment variable is CYCL_HRS (cycle hours) and is set to
-#    the array ("00" "12"), we want the value saved in the local array
-#    here to be a single element consisting of "00 12".  Otherwise, "00" 
-#    and "12" will be treated as separate elements, and more than one 
-#    element would be added to the array (which would be incorrect here).
+#    so that for any experiment variables that are arrays, all the elements of
+#    the array are combined together and treated as a single element.  For
+#    example, if a variable CYCL_HRS is set to the array ("00" "12"), we want
+#    the value saved in the local array here to be a single element consisting
+#    of "00 12".  Otherwise, "00" and "12" will be treated as separate
+#    elements, and more than one element would be added to the array (which
+#    would be incorrect here).
```

Comment:
With the removal of CYCL_HRS, does that mean none of these variables are arrays anymore?

Reply:
I think there could still be arrays elsewhere. I am not sure though. I didn't remove the ability to set an array in the config file, though.